generic-styles/index.less

```diff
@@ -23,24 +23,36 @@ body {
 
 #main-content:not([data-dynamic-style="true"]), .content-excerpt:not([data-dynamic-style="true"]) {
   @import "./content-baked.less";
+  font-size: 1.6rem;
+  line-height: 2.3rem;
 
   // gray is used for text color
   @gray: #424242;
 
+  a {
+    color: #027EB5;
+    cursor: pointer;
+    text-decoration: underline;
+  
+    &:hover {
+      color: #0064A0;
+    }
+  }
+
```

Comment:
@TomWoodward `.page-content` styles moved here. Everything that I've checked looks good.